go.mod

```diff
@@ -2,11 +2,11 @@ module github.com/vmware/terraform-provider-namespace-management
 
 go 1.18
 
-replace github.com/vmware/govmomi => /Users/afowler/Documents/git/govmomi
+// replace github.com/vmware/govmomi => /Users/afowler/Documents/git/govmomi
```

Comment:
Given that `github.com/vmware/govmomi v0.30.0` isn't out, you could consider keeping the replace directive and pointing to your fork of `govmomi`. This is the line that worked for me and allowed me to build against your forked version.

```
replace github.com/vmware/govmomi => github.com/adamfowleruk/govmomi v0.28.1-0.20220720115642-642156dd6e51
```

Benefits of pointing to the fork are that this will build for anyone cloning the repository. The replace directive can then be removed and the required version bumped as soon as `v0.30.0` is released.

Reply:
I've done this in the latest commit. Please review.